### DISCUSSION TITLE
Quantitation Aware Fine Tuning?

### DISCUSSION THREAD

**Discussion by ramkumarkoppu** [keras-team/keras-hub#1075] [General] [answered]
Hi,

I used to do Quantitation aware training on CNN models for improved accuracy on converted TFlite models for int8 inference using tensorflow_model_optimization. Will this same approach useful on keras_nlp LLM fine-tuning for int8 TFlite inference?

**Comment by jbischof** [comment, marked as answer]
Yes @ramkumarkoppu all our models are compatible with TFMOT. See https://www.tensorflow.org/model_optimization
